tutorial/numpy-functions.md

```diff
@@ -0,0 +1,300 @@
+# Supported Numpy function by Pyccel
+
+In Pyccel we try to support the most used Numpy functions by developers. here is some of them:
```

Comment:
Grammar:
- Always use an uppercase letter after a full stop
- `Here is` -> `Here are`

Reply:
> Always use an uppercase letter after a full stop

`here are` -> `Here are`